### PR TITLE
Added bare-bones package.json and .gitignore

### PR DESCRIPTION
**Pull request by huck51**
TODO:
-Add more dependencies
-Add scripts
-peer review

### CODE REVIEW

**Comment by huck51** [comment]
Take a look at this and let me know if I need to make any changes. Also feel free to clone it down and make changes yourself.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "arc_hive",
+  "version": "1.0.0",
```
Comment (agallant) [inline package.json]: May want to set this to 0.0.0 for now - versions are subjective/convention, but often 1.0 means first stable complete release.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "author": "Evan Allen, Tyge Johnson, Patrick Kennedy, Ryan Sherrill, Lois Truby",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/cs1Capstone/arc_hive/issues"
+  },
+  "homepage": "https://github.com/cs1Capstone/arc_hive#readme",
+  "dependencies": {
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "mongoose": "^4.13.5",
```
Comment (agallant) [inline package.json]: Mongo may still make sense for some of your data persisting, but I also encourage checking out Airtable - https://www.npmjs.com/package/airtable

Sorry I've not given specifics yet, I'll try to get a copy of the Airtable base we're actually using and share it with you soon.

**Review by loyslane — APPROVED** [review]
Looks good to me!

**Pull request merged**